src/main/java/session/FormSession.java

```diff
@@ -526,6 +526,7 @@ public FormEntryNavigationResponseBean getNextFormNavigation() throws IOExceptio
         responseBean.setCurrentIndex(currentIndex);
         responseBean.setInstanceXml(null);
         responseBean.setTree(null);
+        responseBean.setStatus(Constants.ANSWER_RESPONSE_STATUS_POSITIVE);
```

Comment:
Is this method only used for SMS? Otherwise I'm a little confused why this would get set here. If you can add a bit of context on what this method does in general, that would probably help (or @dannyroberts if you have enough context on this from reviewing the SMS PR that feel free to ignore me and merge).